src/services/API/User.ts

```diff
@@ -0,0 +1,17 @@
+import { apiClient } from "./RootClient";
+import { externalAuthTokenEndpoint } from "./Endpoints";
+import {
+  UserAPIResponse
+} from "../../types";
+
+export default {
+  /**
+   * authenticates the user using the provided API key
+   * @param {string} apiKey - api key
+   * @returns {Promise<UserAPIResponse>} org details corresponding to the user that is trying to log in
```

Comment:
Remove user from this comment. The quiz doesn't know anything about the link between the user and the org